docs/index.rst

```diff
@@ -30,30 +30,25 @@ On Fedora, you can install these with the following command:
 For python2.x::
 
     dnf install -y gcc git libffi-devel openssl-devel python2-devel \
-        redhat-rpm-config
-    
-    dnf install libxml2-devel
-    
+        redhat-rpm-config libcurl-devel libxml2-devel
+
 For python3.x::
 
     dnf install -y gcc git libffi-devel openssl-devel python3-devel \
-        redhat-rpm-config
-    
-    dnf install libxml2-devel
+        redhat-rpm-config libcurl-devel libxml2-devel
 
 On Red Hat Enterprise Linux 7, you can install these with the following command::
 
     yum install -y gcc git libffi-devel openssl-devel python-devel \
-        redhat-rpm-config
-        
-    yum install libxml2-devel
+        redhat-rpm-config libcurl-devel libxml2-devel
 
 For more information, see `Paramiko: Installing
 <http://www.paramiko.org/installing.html>`_.
 
 Get the source code and install dependencies::
 
     $ git clone git://github.com/SatelliteQE/robottelo.git
+    $ export PYCURL_SSL_LIBRARY=openssl
```

Comment:
@JacobCallahan This will depend on which SSL library libcurl was compiled against. For some environments, it may be
` $ export PYCURL_SSL_LIBRARY=nss` or `$ export PYCURL_SSL_LIBRARY=gnutls`

Reply:
can you elaborate on what condition that might be? I can add a comment to this section to cover that instance.

Reply:
We can add link reference for pycurl installation http://pycurl.io/docs/latest/install.html#ssl

Reply:
For example: 

My RHEL System has NSS based installation : 
```
curl --version
curl 7.29.0 (x86_64-redhat-linux-gnu) libcurl/7.29.0 NSS/3.28.4 zlib/1.2.7 libidn/1.28 libssh2/1.4.3
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smtp smtps telnet tftp 
Features: AsynchDNS GSS-Negotiate IDN IPv6 Largefile NTLM NTLM_WB SSL libz unix-sockets
```
Another fedora test system has openssl based:
```
curl --version 
 curl 7.61.1 (x86_64-redhat-linux-gnu) libcurl/7.61.1 OpenSSL/1.1.1 zlib/1.2.11 brotli/1.0.5 libidn2/2.0.5 libpsl/0.20.2 (+libidn2/2.0.5) libssh/0.8.5/openssl/zlib nghttp2/1.34.0
 Release-Date: 2018-09-05
 Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp 
 Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz brotli TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL Metalink
```